code/datums/crew_objective.dm

```diff
@@ -599,8 +599,8 @@ ABSTRACT_TYPE(/datum/objective/crew/miner)
 		return check_result
 
 
-ABSTRACT_TYPE(/datum/objective/crew/mechanic)
-/datum/objective/crew/mechanic/scanned
+ABSTRACT_TYPE(/datum/objective/crew/engineer)
```

Comment:
Is the abstract type for engineer crew objectives not already defined???

Reply:
Done!